isis/src/control/apps/cnetwinnow/cnetwinnow.cpp

```diff
@@ -90,7 +90,7 @@ namespace Isis {
       }
     }
     //build a Histogram of the residual Magnitudes from the ControlNet
-    Histogram hist(net, &ControlMeasure::GetResidualMagnitude, ui.GetDouble("BIN_WIDTH"));
+    ImageHistogram hist(net, &ControlMeasure::GetResidualMagnitude, ui.GetDouble("BIN_WIDTH"));
```

Comment:
This should use the new Histogram class because it is working with residuals, not pixel values.